backend/models.py

```diff
@@ -1,3 +1,66 @@
-# from django.db import models
+from django.db import models
 
 # Create your models here.
+
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=150)
+
+
+class FieldOfStudy(models.Model):
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
+    name = models.CharField(max_length=150)
+    degree = models.IntegerField()
+
+
+class Yearbook(models.Model):
+    field_of_study = models.ForeignKey(FieldOfStudy, on_delete=models.CASCADE)
+    begin = models.DateField()
+    end = models.DateField()
+    available_capacity = models.IntegerField()
+
+
+class Student(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=50)
+    year_of_exam = models.IntegerField()
+    city = models.CharField(max_length=80)
+
+
+class Graduate(models.Model):
+    student = models.ForeignKey(Student, on_delete=models.CASCADE)
+    yearbook = models.ForeignKey(Yearbook, on_delete=models.CASCADE)
+    grade = models.FloatField()
+
+
+class Recruitment(models.Model):
```

Comment:
Czy to nie powinno być powiązane także z tabelą kandydatów? Bo rozumiem, że to jest tabela, która trzyma proces rekruatcyjny, ale chyba dobrze by było wiedzieć kto aplikował w danym procesie. W sumie co do cykli to chyba działa tak, że można składać tylko raz, ale najwyżej można dostać się w drugiej albo kolejnej turze, więc byśmy musieli zastanowić się jak to zareprezetnować oraz potem wgrywać dane do systemu, kto się zgłosił i z tego wyliczać ile miejsc zostało

Reply:
Jest połączone, ale tabelkę niżej :p

Reply:
Skoro mówisz, że tak zadziała to się nie czepiam :)